NorthstarDLL/core/structs.h

```diff
@@ -18,45 +18,45 @@
 /*
 OFFSET_STRUCT(Name)
 {
-	STRUCT_SIZE(0x100)		// Total in-memory struct size
-	FIELD(0x0, int field)	// offset, signature
+    STRUCT_SIZE(0x100)		// Total in-memory struct size
+    FIELD(0x0, int field)	// offset, signature
 }
 */
 
-#define OFFSET_STRUCT(name) union name
-#define STRUCT_SIZE(size) char __size[size];
-#define STRUCT_FIELD_OFFSET(offset, signature)                                                                                             \
-	struct                                                                                                                                 \
-	{                                                                                                                                      \
-		char CONCAT2(pad, __LINE__)[offset];                                                                                               \
-		signature;                                                                                                                         \
-	};
+#define OFFSET_STRUCT( name ) union name
```

Comment:
What happened to this file?

Reply:
It had `//clang-format off` instead of `// clang-format off` so current clang format still formatted it, while i fixed it here, will go through it and try to reduce the diff